src/databricks/labs/lsql/dashboards.py

```diff
@@ -728,6 +766,118 @@ def _get_query_widget_spec(fields: list[Field], *, frame: WidgetFrameSpec | None
         return spec
 
 
+@dataclass
+class FilterTile(Tile):
+    _position: Position = dataclasses.field(default_factory=lambda: Position(0, 0, 3, 2))
+
+    def validate(self) -> None:
+        """Validate the tile
+
+        Raises:
+            ValueError : If the tile is invalid.
+        """
+        if not self.metadata.is_filter():
+            raise ValueError(f"Tile is not a filter file: {self}")
+        if len(self.metadata.filters) == 0:
+            raise ValueError(f"Filter tile with path {self.metadata.path} has no filters defined")
+
+    def get_layouts(self, datasets: list[Dataset]) -> Iterable[Layout]:
+        """Get the layout(s) reflecting this tile in the dashboard."""
+        dataset_columns = self._get_dataset_columns(datasets)
+        if len(dataset_columns) == 0:
+            err_msg = f"Filter tile with path {self.metadata.path} has no matching dataset columns"
+            raise ValueError(err_msg)
+        widget = self._create_filter_widget(dataset_columns)
+        layout = Layout(widget=widget, position=self.position)
+        yield layout
+
+    def _create_filter_widget(self, dataset_columns: set[tuple[str, str]]) -> Widget:
```

Comment:
Expect the datasets here:

```suggestion
    def _create_filter_widget(self, datasets: list[Dataset]) -> Widget:
```